llvm/tools/repo2obj/repo2obj.cpp

```diff
@@ -69,16 +69,14 @@ class SpecialNames {
 public:
   void initialize(const pstore::database &Db, GeneratedNames &Names);
 
-  pstore::typed_address<pstore::indirect_string> CtorName =
-      pstore::typed_address<pstore::indirect_string>::null();
-  pstore::typed_address<pstore::indirect_string> DtorName =
-      pstore::typed_address<pstore::indirect_string>::null();
+  using istring = pstore::typed_address<pstore::indirect_string>;
+  istring CtorName = istring::null();
+  istring DtorName = istring::null();
 
```

Comment:
Is the code refactoring only here? If yes, could you please put into the different commit.

Reply:
It's adding a typedef (”refactoring” seems like a rather grand term). It was intended to make the code easier to read but if it offends you, I’ll remove it.